submissions/onyinyeChukwuka/HTML/index.html

```diff
@@ -1,71 +1,117 @@
+                <h4>Quantity: <input type="text" value="1" name="quantity " id="quantity"></h4>
+                <h4> Price: &#x20A6 15,000</h4>
+                <button type="submit" id="button"> BUY NOW</button>
+            </div>
+        </div>
+
+        <!-- FOR THE DESCRIPTION -->
+        <div class="description">
+            <h4> DESCRIPTION</h4>
+            <h3> Heavy laced leather boot especially with a wide cuff buckled above the ankle. Suitable for outdoor
+                activities and casual outings.</h3>
+        </div>
     </main>
-    <aside> 
-        <p> 
-            <strong> SIMILAR TO THIS</strong>
-            <br>
-            <ol>
-                <li> <a href="#"> Dr Martens Boots</a></li>
-                <li> <a href="#"> Jay's Trainers </a></li>
-                <li> <a href="#"> Cami's Platform boots </a></li>
-                <li> <a href="#"> Military-style boots </a></li>
-            </ol>
-        </p>
-    </aside>
-    <footer>
-        <p>© 2020 BootStraps</p>
+    <!-- for the related products section -->
+    <div class="related">
+        <h4> RELATED PRODUCTS</h4>
+        <div class="items">
+            <div class="martens">
+                <a href="#"> <img src="../html/images/drmartens.png" alt="Dr-martens-boots" width="399px"
+                        height="298px"></a>
+                <h4>Dr Martens Combat Boots</h4>
+                <h4> Price: &#x20A6 75,000</h4>
+            </div>
+            <div class="platforms">
+                <a href="#"> <img src="../html/images/platform.jpg" alt="Dr-martens-boots" width="399px"
+                        height="298px"></a>
+                <h4> Chain Platform Boots</h4>
+                <h4> Price: &#x20A6 35,000</h4>
+            </div>
+            <div class="military">
+                <a href="#"> <img src="../html/images/military.jpg" alt="Dr-martens-boots" width="399px"
+                        height="298px"></a>
+                <h4>Speedlace Military Boots</h4>
+                <h4> Price: &#x20A6 25,500</h4>
+            </div>
+        </div>
+    </div>
+    <!-- for the footer -->
+    <footer class="footer">
+        <div class="footer-wrapper">
+            <div class="about">
+                <p> Contact Us </p>
+                <p> About Us </p>
+                <p> Terms and Conditions </p>
+            </div>
+
+            <div class="socials">
+                <div class="newsletter">
+                    <label for="email">Sign up for our Newsletter</label><br><br>
+                    <div class="newsletter-input">
+                        <input type="text" id="newslettter" name="email" placeholder="Enter your email address">
+                        <button type="button" id="submitbutton">SUBMIT</button>
+                    </div>
+
+                </div>
+
+                <div class="connect">
+                    <p> CONNECT WITH US</p>
+                    <div class="images">
```

Comment:
Please use a better class name to describe the logos in this footer. `Images` is confusing